metagpt/utils/make_sk_kernel.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/9/13 12:29
+@Author  : femto Zheng
+@File    : make_sk_kernel.py
+"""
+import semantic_kernel as sk
+from semantic_kernel.connectors.ai.open_ai.services.azure_chat_completion import (
+    AzureChatCompletion,
+)
+from semantic_kernel.connectors.ai.open_ai.services.open_ai_chat_completion import (
+    OpenAIChatCompletion,
+)
+
+from metagpt.config import CONFIG
+
+
+def make_sk_kernel():
+    kernel = sk.Kernel()
+    if CONFIG.openai_api_type == "azure":
+        kernel.add_chat_service(
+            "gpt-3.5", AzureChatCompletion(CONFIG.deployment_name, CONFIG.openai_api_base, CONFIG.openai_api_key)
+        )
+    else:
+        kernel.add_chat_service(
+            "gpt-3.5",
```

Comment:
service_id=`gpt-3.5` may cause a confuse about here only use `model gpt-3.5`, suggest to use a more common name like `chat_completion` refs to `https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/kernel/adding-services?tabs=python#openai` or make `service_id` as a param.